roles/kubernetes/node/tasks/install.yml

```diff
@@ -30,7 +30,7 @@
     - container_manager == "docker"
 
 - name: install | Copy socat wrapper for Container Linux with Containerd
-  command: "{{ containerd_bin_dir }}/ctr run --rm --mount type=bind,src={{ bin_dir }},dst=/opt/bin,options=rbind:rw {{ install_socat_image_repo }}:{{ install_socat_image_tag }}"
+  command: "{{ containerd_bin_dir }}/ctr --namespace k8s.io run --rm --mount type=bind,src={{ bin_dir }},dst=/opt/bin,options=rbind:rw {{ install_socat_image_repo }}:{{ install_socat_image_tag }} copysocat"
```

Comment:
Why do we need to change here for supporting Flatcar  Container Linux?

Reply:
There are two reason.
 1. ```ctr run``` needs container id. Error message is ```ctr: container id must be provided``` 
 2. container load to ```k8s.io``` namespace. Error message is ```ctr: image "docker.io/xueshanf/install-socat:latest": not found'''